### PR TITLE
Use categories

### PR DESCRIPTION
**Pull request by alexandrelissowski**
Add an article to explain how to use categories for suppliers.

### CODE REVIEW

File: content/md/supplier-onboarder/handle-category.md [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+## View the categories of a product
+### In the Product Form
+
+1.  Click on the `Categories` menu
+1.  A green point near the Category tree indicates if the product is classified in this tree. Next to the category tree name, a number tells you in how many categories the product has been added
+1.  Click on the category tree name to see the categories the products belong to
+
+Selected categories are ticked for your product.
+
+![Categories menu](../img/Products_PEF_Category-menu.png)
```
Comment (felix-couprie) [inline content/md/supplier-onboarder/handle-category.md:74]: We can see your name and surname here (but it's maybe not an issue for you)

File: content/md/supplier-onboarder/handle-category.md [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+1.  Hover your mouse over the product line in the grid to display the category button (a folder icon) will be displayed
+1.  Click on the `Classify the product` button. You are redirected to the [category menu](/handle-category.html#in-the-product-form).
+![Hover grid](../img/Products_GridHover_Category.png)
+
+
+## Classify/unclassify a product in/from categories
+### With the user interface
+
+To classify/unclassify a product:
+1.  Select your product in the product grid Product
```
Comment (felix-couprie) [inline content/md/supplier-onboarder/handle-category.md:88]: Typo? "product grid Product"

File: content/md/supplier-onboarder/handle-category.md [hunk 3]
```diff
@@ -0,0 +1,102 @@
...
+1.  Click on the `Categories` menu on the left side of the screen
+1.  Tick the categories in which you want to classify the product
+1.  And/or untick the categories in which the product should no longer be listed
+1.  Then click on the "Save" button
+
+### With imports
+
+You can also update your products' categories using product imports. To do so, add the category code in the `Categories column` used to import categories.
```
Comment (felix-couprie) [inline content/md/supplier-onboarder/handle-category.md:96]: Separated by commas if multiple categories

**Review by felix-couprie — APPROVED** [review]
Few comments and go :)

File: content/md/supplier-onboarder/handle-category.md (hunk 2 above)
Comment (alexandrelissowski) [reply content/md/supplier-onboarder/handle-category.md:88]: typo

File: content/md/supplier-onboarder/handle-category.md (hunk 3 above)
Comment (alexandrelissowski) [reply content/md/supplier-onboarder/handle-category.md:96]: thanks

**Comment by alexandrelissowski** [comment]
English request

- Name: Alexandre Lissowski
- Wordcount: 600
- Deadline:
- T-shirt size: S

**Comment by nestorAkeneo** [comment]
Your proofreading request has been created

Ticket link: https://akeneo.atlassian.net/browse/ENG-95:

File: content/md/supplier-onboarder/handle-category.md [hunk 4]
```diff
@@ -0,0 +1,102 @@
+---
+id: handle-category
+themes: manage-products
+title: Handle **categories**
+popular: false
+---
+
+# Definition of a category
```
Comment (camillebourjade) [inline content/md/supplier-onboarder/handle-category.md:8]: What is a category?

File: content/md/supplier-onboarder/handle-category.md [hunk 5]
```diff
@@ -0,0 +1,102 @@
...
+In Akeneo Onboarder, you can use **a category to classify products**. A category is always part of a category tree (or *classification tree*).
+
+As an Onboarder user, you can only have **one category tree** in Akeneo with an **unlimited number of levels** (categories, subcategories, sub-subcategories).
+
+![image of the category trees](../img/Settings_CategoryTree.png)
+
+## Differences between families and categories
+In this section, we define two principles of the Akeneo Onboarder: *Categories* and *Families*.
```
Comment (camillebourjade) [inline content/md/supplier-onboarder/handle-category.md:16]:
```suggestion
In this section, we define two key concepts of the Akeneo Onboarder: *Categories* and *Families*.
```

Comment (camillebourjade) [reply content/md/supplier-onboarder/handle-category.md:16]: maybe think about this structure
- def cat
- def fam 
- difference cat Vs fam

File: content/md/supplier-onboarder/handle-category.md [hunk 6]
```diff
@@ -0,0 +1,102 @@
...
+### Categories
+
+- Trees and categories are a way to classify your products  
+- A product can be in several or no categories
```
Comment (camillebourjade) [inline content/md/supplier-onboarder/handle-category.md:20]: active form -> you can classify a product in one or several categories. It's is possible for a product to not be in a category, in that case, it goes to the 'Unclassified' Category by default.

File: content/md/supplier-onboarder/handle-category.md [hunk 7]
```diff
@@ -0,0 +1,102 @@
...
+
+### Families
+
+- A family is a *set of attributes* used to enrich a product  
+- A product can only belong to one family
```
Comment (camillebourjade) [inline content/md/supplier-onboarder/handle-category.md:25]:
```suggestion
- A product can only be defined by one family and its specific set of attributes. 
```

Comment (camillebourjade) [reply content/md/supplier-onboarder/handle-category.md:25]: Maybe use that for the definition

File: content/md/supplier-onboarder/handle-category.md [hunk 8]
```diff
@@ -0,0 +1,102 @@
...
+
+## Note about category visibility
+The category tree is **not** visible in the Akeneo PIM of you retailer or distributor. But for each product you create, the category is displayed for [new product proposal](/onboarder/articles/supplier-synchronization.html) into the PIM.
```
Comment (camillebourjade) [inline content/md/supplier-onboarder/handle-category.md:28]:
```suggestion
The category tree you are going to generate will **not** be visible in the Akeneo PIM of your retailer or distributor. But for each product you create, the category name is included and visible in the [new product proposal](/onboarder/articles/supplier-synchronization.html). Therefore, the Akeneo PIM users, on the retailer or distributor side, will be able to see it.
```

File: content/md/supplier-onboarder/handle-category.md [hunk 9]
```diff
@@ -0,0 +1,102 @@
...
+# Manage your categories
+## How to create a new category?
+
+Go to the `Categories` tab to manage all your categories.
+
+::: info
+You can only create **one** category tree in the Onboarder.
+:::
+
+To add a new category or subcategory in your category tree:
```
Comment (camillebourjade) [inline content/md/supplier-onboarder/handle-category.md:39]: reformule! 
step 1 
CAT A 
step 2 
subcat1
subct2
Step 3 
CAT B from cat A

**Review by camillebourjade — COMMENTED** [review]
English fix

**Review by camillebourjade — COMMENTED** [review]
English Fix + comments

File: content/md/supplier-onboarder/handle-category.md [hunk 10]
```diff
@@ -0,0 +1,118 @@
...
+
+
+
+## Differences between families and categories
+In this section, we'll take an example to compare two key concepts of the Akeneo Onboarder: *Families* and *Categories*.
+
+Our catalog contains lots of furnitures: chairs and tables for instance.
+On the one hand, for chairs you'll define: heigh, material, seat depth, foam softness, etc, And, for table you'll define heigh, length, with of the board, number of feet, etc. You see that the two products don't have the same properties, it means that the sets of attributes are different. Finally, they belong to separated **families**.
+
+On the other hand, for all kind of furnitures, you can set them in the indoor or outdoor section of your catalog by using a **category**. In our example, the chairs and tables can be one, both or none of the categories.
```
Comment (alexandrelissowski) [inline content/md/supplier-onboarder/handle-category.md:33]: @camillebourjade following your suggestion, I added an example here.

File: content/md/supplier-onboarder/handle-category.md [hunk 11]
```diff
@@ -0,0 +1,118 @@
...
+
+![image of the category trees](../img/Settings_CategoryTree.png)
+
+
+
+## Differences between families and categories
+In this section, we'll take an example to compare two key concepts of the Akeneo Onboarder: *Families* and *Categories*.
+
+Our catalog contains lots of furnitures: chairs and tables for instance.
+On the one hand, for chairs you'll define: heigh, material, seat depth, foam softness, etc, And, for table you'll define heigh, length, with of the board, number of feet, etc. You see that the two products don't have the same properties, it means that the sets of attributes are different. Finally, they belong to separated **families**.
```
Comment (camillebourjade) [inline content/md/supplier-onboarder/handle-category.md:31]:
```suggestion
First of all, for chairs, you'll define the height, the fabric, seat depth, the foam density, etc. For tables, you'll define the height, the length, the width of the board, the number of feet, etc.  As you can see, you use different properties to describe both products, which means that the sets of attributes you will use will be different. In the end, they belong to two different **families**.
```

File: content/md/supplier-onboarder/handle-category.md (hunk 10 above)
Comment (camillebourjade) [reply content/md/supplier-onboarder/handle-category.md:33]: Then, all of your furniture can belong to the indoor or outdoor section of your catalog, which is structured by a **category**. In our example, the chairs and tables can be one, both, or none of the categories.

**Review by camillebourjade — COMMENTED** [review]
English fix

**Pull request merged**